### DISCUSSION TITLE
USDC from Solana to Eth is lost

### DISCUSSION THREAD

**Discussion by SSSemtex** [wormhole-foundation/wormhole#461] [Problems / bugs] [answered]
I tried to bridge USDC from Solana to Eth and it appears to have gone missing. After trying to complete the intitiation manually, I still cant seem to see where its gone.

Solana tx: 2HqiCwCz75MPs7ZFNx9w4HdZNy2pNamMKXtLfArVjQMhGvZ9vKDCinKRKYkKyQWg5GQka2nUmvLe16GonyR6JtLW

Ethereum confimation: 0x51621b63e738186093f5716b4cc9edf616eb7c273fdfc2a2d954a4ce77bfc2a7

Any help would be greatly appreciated.

**Comment by quentinadam** [comment, marked as answer]
Hello @SSSemtex 

Disclaimer: I am not a Wormhole developper but a regular user.

Shortly put, the Wormhole bridge cannot be used to bridge USDC from Solana to Ethereum. You will end up, on the Ethereum blockchain, with an asset that is not USDC, but a Wormhole wrapped version of it. As a matter of fact you did receive 1035 of these Wormhole wrapped USDC tokens on Ethereum (token address https://etherscan.io/token/0x41f7b8b9b897276b7aae926a9016935280b44e97).

What I suggest you to do is to use the Wormhole bridge in the other direction to convert back the wormhole wrapped USDC tokens from Ethereum to USDC on Solana.

Once you have the USDC on Solana, one way to bridge your USDC from Solana to Ethereum is through the exchange FTX.com.

**Reply by leoluk** [reply]
This is the correct answer, thanks!

**Reply by quentinadam** [reply]
@leoluk Glad I could help. Did you have a chance to check my recent question (https://github.com/certusone/wormhole/discussions/455) ?

**Reply by leoluk** [reply]
Not yet
